src/main/java/com/idrsolutions/microservice/BaseServlet.java

```diff
@@ -396,6 +396,13 @@ private boolean handleFileFromUrl(final Individual individual, final HttpServlet
             doError(request, response, "No url given", 400);
             return false;
         }
+
+        String protocol = url.substring(0, url.indexOf(":"));
```

Comment:
If we send a URL without a protocol this could throw an exception as the end index would be -1.
I suggest we use starts with http:// and https://, we should also probably use equalsIgnoreCase